src/js/api/brand.js

```diff
@@ -142,14 +142,15 @@ ripe.Ripe.prototype.getFontUrl = function(name, format, options) {
 /**
  * Returns the URL for the environment file of the 3D scene.
  *
- * @param {Object} options A map with options, such as:
- *  - 'brand' - The brand of the model.
- *  - 'model' - The name of the model.
- *  - 'version' - The version of the build, defaults to latest.
+ * @param {String} name The name of the environment map.
+ * @param {String} format The file format of the environment map.
+ * @param {Object} options A map with options.

```

Comment:
```suggestion
 * @param {String} 'name' - The name of the environment map.

```

Use same format as others

Reply:
The format is correct, check https://github.com/ripe-tech/ripe-sdk/blob/563ba2394316f4ce2fdb6ecbed7f818afa246997/src/js/api/config.js#L118

I think you are confusing with the documentation of object params.